### PR TITLE
Update UserDetailsFrom style for escrow verification flow

### PR DESCRIPTION
**Pull request by devinjameson**
Why: we would like for the user details form to provide a good UX on both iOS and Android.

### CODE REVIEW

File: src/EscrowVerification/UserDetailsForm.tsx [hunk 1]
```diff
@@ -45,20 +47,26 @@ export const phoneToFormattedString = (phonenumber: string): string => {
...
+  const [isPhoneInputFocused, setIsPhoneInputFocused] = useState(false)
   const [errorMessage, setErrorMessage] = useState(defaultErrorMessage)
+  const [showDatePickerAndroid, setShowDatePickerAndroid] = useState(false)
 
-  const handleOnChangeTestDate = (testDate: Date | undefined) => {
+  const handleOnChangeTestDate = (
+    _event: Event,
+    testDate: Date | undefined,
+  ) => {
+    setShowDatePickerAndroid(false)
```
Comment (devinjameson) [inline src/EscrowVerification/UserDetailsForm.tsx:69]: This line seems weird, but there's a really UI bug that occurs if it's not included. Found in the issues for the date picker library.

Comment (johnschoeman) [reply src/EscrowVerification/UserDetailsForm.tsx:69]: Could we make sure to mention this in the commit message?

**Review by johnschoeman — APPROVED** [review]
LGTM!

File: src/EscrowVerification/UserDetailsForm.tsx (hunk 1 above)
Comment (johnschoeman) [reply src/EscrowVerification/UserDetailsForm.tsx:69]: and add the issue link here as well?

Comment (johnschoeman) [reply src/EscrowVerification/UserDetailsForm.tsx:69]: https://github.com/react-native-datetimepicker/datetimepicker/issues/263
https://github.com/react-native-datetimepicker/datetimepicker/issues/54

**Pull request merged**